config/routes.rb

```diff
@@ -168,6 +168,8 @@
     scope module: :projects do
       resources :projects, only: [:show] do
         resource :extract, only: [:show]
+        resource :classify, only: [:show]
```

Comment:
Should this not be part of the existing classifications controller, perhaps renamed to classify?

Reply:
I added a note about this in the commit message